share/picongpu/examples/LaserWakefield/lib/python/picongpu/params.py

```diff
@@ -36,6 +39,12 @@
 
     LinearScaledParameter(
         name="Pulse_Length_SI", ptype="compile", unit="s",
-        default=5, slider_min=1, slider_max=150,
+        default=5.0, slider_min=1.0, slider_max=150.0,
         slider_step=1, scale_factor=1.e-15),
+
+    LinearScaledParameter(
+        name="TBG_steps", ptype="run", unit="s",
```

Comment:
It would probably be less confusing to use the units the user is accustomed to like pico-seconds, nano-meters etc as units. The formatted internal picongpu value of each parameter is, as discussed yesterday, indeed not relevant to the user. Sorry for switching back and forth about this issue.